src/ApprenticeAan/SFA.DAS.ApprenticeAan.Api.UnitTests/Models/GetCalendarEventsQueryTests.cs

```diff
@@ -0,0 +1,49 @@
+﻿using AutoFixture.NUnit3;
+using FluentAssertions;
+using SFA.DAS.ApprenticeAan.Api.Models;
+using SFA.DAS.ApprenticeAan.Application.CalendarEvents.Queries.GetCalendarEvents;
+using SFA.DAS.ApprenticeAan.Application.Common;
+
+namespace SFA.DAS.ApprenticeAan.Api.UnitTests.Models;
+public class GetCalendarEventsRequestModelTests
+{
+    [Test, AutoData]
+    public void Operator_PopulatesQueryFromModel(Guid memberId, DateTime? fromDate, DateTime? toDate, List<EventFormat> eventFormats, List<int> calendarIds, List<int> regionIds, int? page, int? pageSize)
+    {
+        var model = new GetCalendarEventsRequestModel
+        {
+            RequestedByMemberId = memberId,
+            FromDate = fromDate,
+            ToDate = toDate,
+            EventFormat = eventFormats,
+            CalendarId = calendarIds,
+            RegionId = regionIds,
+            Page = page,
+            PageSize = pageSize
+        };
+
+        var query = (GetCalendarEventsQuery)model;
+
+        query.RequestedByMemberId.Should().Be(memberId);
+        query.FromDate.Should().Be(fromDate?.ToString("yyyy-MM-dd"));
+        query.ToDate.Should().Be(toDate?.ToString("yyyy-MM-dd"));
+        query.EventFormat.Should().BeEquivalentTo(eventFormats);
+        query.Page.Should().Be(page);
+        query.PageSize.Should().Be(pageSize);
+    }
+
+    [Test, AutoData]
+    public void Operator_PopulatesQueryFromModelWithNulls(Guid memberId)
+    {
+        var model = new GetCalendarEventsQuery
```

Comment:
I think this is still not correct as we are not really testing anything here. Ideally I would expect that you populate member id in the model and then do the conversion from model to query and then all your asserts would make sense.